### ISSUE TITLE
Add data from Manni et al. 2016

### ISSUE THREAD

**Issue by SimonGreenhill** [glottobank/lexibank-data#32]
https://www.researchgate.net/publication/303865336_A_Central_Asian_Language_Survey_Collecting_Data_Measuring_Relatedness_and_Detecting_Loans

`We have documented language varieties (either Turkic or Indo-European) spoken in 23 test sites by 88 informants belonging to the major ethnic groups of Kyrgyzstan, Tajikistan and Uzbekistan (Karakalpaks, Kazakhs, Kyrgyz, Tajiks, Uzbeks, Yaghnobis). The recorded linguistic material concerns 176 words of the extended Swadesh list and will be made publically available with the publication of this paper. Phonological diversity is measured by the Levenshtein distance and displayed as a consensus bootstrap tree and as multidimensional scaling plots. Linguistic contact is measured as the number of borrowings, from one linguistic family into the other, according to a precision/recall analysis further validated by expert judgment. Concerning Turkic languages, the results of our sample do not support Kazakh and Karakalpak as distinct languages and indicate the existence of several separate Karakalpak varieties. Kyrgyz and Uzbek, on the other hand, appear quite homogeneous. Among the Indo-Iranian languages, the distinction between Tajik and Yaghnobi varieties is very clear-cut. More generally, the degree of borrowing is higher than average where language families are in contact in one of the many sorts of situations characterizing Central Asia: frequent bilingualism, shifting political boundaries, ethnic groups living outside the “mother” country. `

**Comment by LinguList** [comment]
I have this data in a cleaned segmented form, from G. Jäger, who got the
data before official publication. However, officially, we'll have to
wait until the paper is really out and LDC publishes it with
supplementary material. Anyway, one of Gerhard's students linked most
concepts to the concepticon, and I helped further by spotting coding
errors and lingpy-compatibility, and we will safe some work when
checking back from there.

**Comment by LinguList** [comment]
Just saw that the data is now officially published at figshare:

https://figshare.com/articles/A_Central_Asian_Language_Survey/3443090/1

**Comment by LinguList** [comment]
But they make me actually a bit sad, as this is not what I understand as supplementary material. It is a word document, well ordered in tables, with cognate codes, etc. but no way to automatically make it into a spreadsheet or similar.

**Comment by xrotwang** [comment]
Parsing well formatted word tables is actually quite easy. I'll have a look
on monday.
Am 27.08.2016 20:33 schrieb "Johann-Mattis List" <notifications@github.com>:

**Comment by xrotwang** [comment]
@LinguList do we want to put their adapted Swadesh 207 list into concepticon? I'd say yes, in particular because they provide explanations why and how they changed it.

**Comment by xrotwang** [comment]
Parsing the tables in the word file was straightforward, resulting in 23 csv files with 87 rows each. The whole dataset seems to be in the range of 18,000 lexemes.

**Comment by xrotwang** [comment]
With a bit of fiddling around I can now even extract the background color of the table cells :)
```python
colorp = re.compile('fill\="(?P<color>[^"]+)"')


def text_and_color(cell):
    color = None
    for line in cell._tc.tcPr.xml.split('\n'):
        if 'w:shd' in line:
            m = colorp.search(line)
            if m:
                color = m.group('color')
                break
    if color == 'auto':
        color = None
    return cell.paragraphs[0].text, color
```

**Comment by xrotwang** [comment]
@LinguList the 207-item Swadesh list they talk about in the paper seems not to be in Concepticon as well.

**Comment by LinguList** [comment]
it is the list by Comrie-197?-207. At least, Comrie was the first to merge Swadesh-1955-100 and Swadesh-1952-200 into 207 (also in Wiktionary). I need to re-read the paper. They do not seem to be too informed on the exact history of the concept list history. 

This also relates to the aliases, as the 207 list is a fiction in the literature, often quoted, but hard to trace, where people get it exactly (with Comrie being the earliest source).

**Comment by xrotwang** [comment]
Ah, I see. This list almost matches - of course not exactly, though :)

**Comment by LinguList** [comment]
not surprised by that ;-), it sometimes reminds me of playing "Stille Post" as a kid. One time, I want to make a stemmatological analysis of all concept lists.

**Comment by xrotwang** [comment]
One question we have to answer for this dataset is how to handle data linked to informants, rather than varieties. Treat each informant as variety? Or treat differing forms from informants speaking the same language as variants for one language? For this dataset, I'm leaning to the latter approach, because otherwise we'd get a lot of seemingly duplicate data.

**Comment by LinguList** [comment]
I completely agree. we could make a precedence for cldf format here: As we discussed one time in the past ranked variation, say German "kill" has "umbringen" and "töten", but we'd add a rank-column in cldf indicating that "umbringen" is more frequently used. This data seems to be the ideal case to test for ranking, based on frequency, and using some alignment could help to get some order.

**Comment by SimonGreenhill** [comment]
One thing I'm concerned about is that we don't want to rule out dialect-level datasets right? the precedence for token idea would make it much harder to add these into lexibank. What about adding these as some form of 'doculect' for the same language?

**Comment by xrotwang** [comment]
I guess adding each informant as doculect is more consistent, and also simpler in terms of interpreting and importing the data. Also condensing the data later, turning multiple occurrences into frequencies is still possible. The only downside really is that browsing data with many "duplicates" looks weird.

**Comment by SimonGreenhill** [comment]
I agree, but it's a reality of these datasets e.g. this one in tng.org (http://transnewguinea.org/source/fleischmann-and-turpeinen-1976?sort=word). What I did in tng.org was to give each of these doculects its own dialect.

**Comment by xrotwang** [comment]
Ok, good point. So we basically already have a policy for handling this: Add varieties/doculects. Actual dialects may be disambiguated later via finer-grained Glottocodes, doculects maybe as well, informants rather not.

**Comment by SimonGreenhill** [comment]
yes, I'm reluctant to go to 'informant' level data but I think this won't be a common type of data. Varieties/doculect level datasets are everywhere and they will be cool to have.

**Comment by LinguList** [comment]
I'd opt for one thing in this concept: Don't have languages = glottocodes as primary sorter, but language-varieties, and people can sort by glottocode. This makes it much easier to inspect, e.g., the data on Chinese, but when adding dialects, e.g., Bulgarian (in BDPA) and others, we can even do more.
